### PR TITLE
Use ruleset.xml by default if present

### PR DESCRIPTION
**Pull request by filipesperandio**

### CODE REVIEW

File: src/Config.groovy [hunk 1]
```diff
@@ -34,6 +34,11 @@ class Config {
       }
     }
 
+    def prjRulesByConvention = new File(appContext.codeFolder, "ruleset.xml")
```
Comment (toddmazierski) [inline src/Config.groovy]: Is `prj` an abbreviation for `project`? Is it a common one in Groovy? If not, I'd suggest using the full word instead.

File: test/ConfigTest.groovy [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+  @Test
+  public void specifiedRuleSetFile() {
+    def config = new Config([configFile: "/usr/src/app/fixtures/config.rule.json", codeFolder: "/usr/src/app/fixtures"])
+    assertEquals config.ruleSet(), "/usr/src/app/fixtures/specified_rules.xml"
+  }
+
+  @Test
+  public void honorPresentRules() {
+    def config = new Config([configFile: "/usr/src/app/fixtures/config.json", codeFolder: "/usr/src/app/fixtures/honor"])
```
Comment (toddmazierski) [inline test/ConfigTest.groovy]: What does the name `/honor` mean in the path? Is this a fixture directory just for this test?

If yes, I might suggest something more explicit like `/ruleset_default`, and maybe moving `specified_rules.xml` and `config.rule.json` to something like `/ruleset_specified`.

Comment (filipesperandio) [reply test/ConfigTest.groovy]: @toddmazierski Re-structured `fixture` directory. Do you mind giving another shot?

**Comment by filipesperandio** [comment]
@toddmazierski I liked your suggestion about the `fixtures`, the final structure is much easier to understand:
```
fixtures/
├── bad_config
│   ├── Main.java
│   └── config.json
├── default
│   ├── Main.java
│   └── config.json
├── ruleset_default_file
│   ├── Main.java
│   ├── config.json
│   └── ruleset.xml
└── specified_file
    ├── Main.java
    ├── config.new.json
    ├── config.old.json
    └── rules.xml
```

**Pull request merged**